support/agenda-queries.js

```diff
@@ -0,0 +1,90 @@
+import {query, sparqlEscapeString} from "mu";
+import {prefixMap} from "./prefixes";
+
+/**
+ * @typedef {import("./types").Support}
+ */
+
+/**
+ * Retrieves the zitting belonging to the supplied zitting uuid
+ *
+ * @method editorDocumentFromUuid
+ *
+ * @param {string} uuid UUID which is coupled to the Zitting
+ * mu:uuid property.
+ *
+ * @return {Promise<Support.Zitting>} Promise which resolves to an object representing
+ * the zitting
+ */
+async function getZitting(uuid) {
+  /**
+   * @typedef { "uri"
+   * | "agendapunten"
+   * | "bestuursorgaan"
+   * | "geplandeStart"
+   * | "gestartOpTijdstip"
+   * | "geeindigdOpTijdstip" } QVars
+   */
+  /**
+   * @type {Support.QueryResult<QVars>}
+   */
+  const queryResult = await query(
+    `${prefixMap.get("ext").toSparqlString()} 
+     ${prefixMap.get("besluit").toSparqlString()}
+     ${prefixMap.get("prov").toSparqlString()}
+     SELECT * WHERE {
+       ?uri a besluit:Zitting;
+            besluit:behandelt ?agendapunten;
+            besluit:isGehoudenDoor ?bestuursorgaan;
+            besluit:geplandeStart ?geplandeStart;
+            prov:startedAtTime ?gestartOpTijdstip;
```

Comment:
startedAtTime and endedAtTime are typically not filled in when the agenda is published, but only when the meeting has happened.